tests/test_scan.py

```diff
@@ -0,0 +1,19 @@
+from unittest import TestCase
+
+from wifi.scan import Cell
+from wifi import subprocess_compat as subprocess
+from mock import MagicMock
+from test_parsing import output
+
+class ScanTest(TestCase):
+    def test_all_calls_check_output_with_good_args(self):
+        args = ['sudo', '/sbin/iwlist', 'interface', 'scan']
+        kwargs = {'stderr':subprocess.STDOUT}
+        subprocess.check_output = MagicMock(return_value=output)
+        Cell.all('interface', sudo=True)
+        subprocess.check_output.assert_called_once_with(args, **kwargs)
+        args = ['/sbin/iwlist', 'interface', 'scan']
```

Comment:
If you do it the other way around, you can set args only once, and then inserting `sudo` for next test.